### PR TITLE
feat: add details to each resource of an appeal

### PR DESCRIPTION
**Pull request by ishanarya0**
Added a details field to each of the resources of an appeal.

### CODE REVIEW

**Comment by ishanarya0** [comment]
Changed the service name from Guardian to GuardianService, as in current use by Guardian repo.

https://github.com/odpf/guardian/blob/9a4200f173b66b18261b79be31f8b09407927fe8/api/proto/odpf/guardian/guardian.pb.gw.go#L34

File: odpf/guardian/guardian.proto [hunk 1]
```diff
@@ -23,7 +23,7 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
 };
 
 // WARNING: This is still in active development and can have breaking changes
-service Guardian {
+service GuardianService {
```
Comment (rahmatrhd) [inline odpf/guardian/guardian.proto:26]: let's keep it as `Guardian`

**Review by rahmatrhd — CHANGES_REQUESTED** [review]

File: odpf/guardian/guardian.proto (hunk 1 above)
Comment (ishanarya0) [reply odpf/guardian/guardian.proto:26]: Sure, but then we need to update it in the Guardian repo as well.

Comment (rahmatrhd) [reply odpf/guardian/guardian.proto:26]: yes, will update from an upcoming PR

Comment (rahmatrhd) [reply odpf/guardian/guardian.proto:26]: cancelling this suggestion as per this [previous comment](https://github.com/odpf/proton/pull/17#discussion_r670766457)

**Review by rahmatrhd — APPROVED** [review]

**Pull request merged**